<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComp</class>
 <widget class="QDialog" name="AddComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add computers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_errorName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="button_built">
           <property name="text">
            <string>Built</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="button_notbuilt">
           <property name="text">
            <string>Not built</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lebel_buildyear">
         <property name="text">
          <string>Build Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_errorYear">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="line_build_year">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_type">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_type">
         <item>
          <property name="text">
           <string>Electronic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mechanical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transistor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_add">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_addsuccess">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_notbuilt</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_build_year</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
